data/RoHit4100.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Rohit Udamale",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "**Student**, Currently learning web3 and react, Want to contribute to real life projects",
+  "avatar": "https://avatars.githubusercontent.com/u/70364267?s=400&u=6ae48f02ca4bf044f149a8d33630d3f1fa117e65&v=4",
```

Update image url to short form.
```suggestion
  "avatar": "https://github.com/RoHit4100.png",
```